bfh/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.3.0"
+__version__ = "0.3.1"
```

Comment:
patch is for bugfixes. bump the minor. http://semver.org

also please final newline do